Master unittests are failing on Windows

Latest master unit tests are failing on Windows 8, with the following error:
```
  1) globUtil resolveFileGlobPatterns() should convert a directory name with a leading slash into a glob pattern:

      AssertionError: expected [ '/lib' ] to deeply equal [ '/lib/**/*.js' ]
      + expected - actual

       [
      -  "/lib"
      +  "/lib/**/*.js"
       ]

      at Context.<anonymous> (c:\Users\Ilya\Documents\github\eslint\tests\lib\util\glob-util.js:78:20)
```
CC @IanVS 

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/27883971-master-unittests-are-failing-on-windows?utm_campaign=plugin&utm_content=tracker%2F282608&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F282608&utm_medium=issues&utm_source=github).
</bountysource-plugin>

Between path issues and line ending issues, I'm wondering if we should try
to get a Windows CI environment up.

That said, I've checked Travis and they're still a ways away from Windows
support...
On Nov 1, 2015 11:44 PM, "Ilya Volodin" <notifications@github.com> wrote:

> Latest master unit tests are failing on Windows 8, with the following
> error:
>
>   1) globUtil resolveFileGlobPatterns() should convert a directory name with a leading slash into a glob pattern:
>
>       AssertionError: expected [ '/lib' ] to deeply equal [ '/lib/**/*.js' ]
>       + expected - actual
>
>        [
>       -  "/lib"
>       +  "/lib/**/*.js"
>        ]
>
>       at Context.<anonymous> (c:\Users\Ilya\Documents\github\eslint\tests\lib\util\glob-util.js:78:20)
>
> CC @IanVS <https://github.com/IanVS>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eslint/eslint/issues/4315>.
>

--- a/tests/lib/util/glob-util.js
+++ b/tests/lib/util/glob-util.js
@@ -71,13 +71,6 @@ describe("globUtil", function() {
             assert.deepEqual(result, ["lib/**/*.js", "tests/**/*.js"]);
         });
 
-        it("should convert a directory name with a leading slash into a glob pattern", function() {
-            var patterns = ["/lib"];
-            var result = globUtil.resolveFileGlobPatterns(patterns);
-
-            assert.deepEqual(result, ["/lib/**/*.js"]);
-        });
-
         it("should remove leading './' from glob patterns", function() {
             var patterns = ["./lib"];
             var result = globUtil.resolveFileGlobPatterns(patterns);

